packages/modal/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "pcln-modal",
+  "version": "1.0.0",
+  "description": "React component for modal",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "jest",
+    "prepare": "rollup --config",
+    "clean": "rm -rf node_modules dist"
+  },
+  "author": "Priceline",
+  "license": "MIT",
+  "dependencies": {
+    "@reach/dialog": "^0.1.2",
+    "react-spring": "^6.1.6"
```

Comment:
@bertya one thing I ran into when I forked this modal for use in our app was that `react-spring` was breaking the bamboo build on `object.entries` since we are using node6 for prod. I ended up removing this dep/animation for now - if we want to use it I think a polyfill may be required.